src/signals/incident/definitions/wizard-step-2-vulaan/wegen-verkeer-straatmeubilair.js

```diff
@@ -25,7 +25,7 @@ export const controls = {
   extra_kerstverlichting: {
     meta: {
       label: '',
```

Comment:
Although not part of this PR, I missed these things in the previous PR for this story: 

- Label key can be left out altogether
- `['lantaarnpaal-straatverlichting']` can be `'lantaarnpaal-straatverlichting'`

Reply:
Good point, I changed this as wel!